packages/hagrid/hagrid/cli.py

```diff
@@ -1388,13 +1570,39 @@ def create_launch_cmd(
     )
 
 
+def pull_command(cmd: str, kwargs: TypeDict[str, Any]) -> TypeList[str]:
+    pull_cmd = str(cmd)
+    if kwargs["release"] == "production":
+        pull_cmd += " --file docker-compose.yml"
+    else:
+        pull_cmd += " --file docker-compose.pull.yml"
+    pull_cmd += " pull"
+    return [pull_cmd]
+
+
+def build_command(cmd: str) -> TypeList[str]:
+    build_cmd = str(cmd)
+    build_cmd += " --file docker-compose.build.yml"
+    build_cmd += " --file docker-compose.dev.yml"
+    build_cmd += " build"
+    return [build_cmd]
+
+
+def deploy_command(cmd: str, tail: bool) -> TypeList[str]:
```

Comment:
when I execute this hagrid command
`!hagrid launch to docker:8082 --dev --silent`
The launch progress freezes, maybe an error on the rich library, not sure

![Screenshot 2022-10-31 at 9 37 25 AM](https://user-images.githubusercontent.com/43314053/199030286-145df063-0e1a-44ed-87bd-23e98bbc3c4f.png)